### PR TITLE
*: use `explain format = 'brief'` for tests

### PR DESCRIPTION
**Pull request by eurekaka**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/22663

Problem Summary:
Do not print ID in the explain output of test cases.

### What is changed and how it works?

What's Changed:

Use new syntax `explain format = 'brief'` for a simpler plan output, which is more stable.

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

Side effects

N/A

### Release note 

- N/A

### CODE REVIEW

File: planner/core/plan.go [hunk 1]
```diff
@@ -601,6 +601,9 @@ func (p *basePlan) ExplainInfo() string {
 
 func (p *basePlan) ExplainID() fmt.Stringer {
 	return stringutil.MemoizeStr(func() string {
+		if p.ctx != nil && p.ctx.GetSessionVars().StmtCtx.IgnoreExplainIDSuffix {
+			return p.tp
+		}
```
Comment (eurekaka) [inline planner/core/plan.go:606]: This is the main code change of this PR.

**Review by time-and-fate — COMMENTED** [review]
LGTM

**Review by qw4990 — APPROVED** [review]
LGTM

**Pull request merged**